cmd/install/install.go

```diff
@@ -67,6 +69,7 @@ func NewCommand() *cobra.Command {
 	cmd.Flags().BoolVar(&opts.ExcludeEtcdManifests, "exclude-etcd", false, "Leave out etcd manifests")
 	cmd.Flags().BoolVar(&opts.EnableOCPClusterMonitoring, "enable-ocp-cluster-monitoring", opts.EnableOCPClusterMonitoring, "Development-only option that will make your OCP cluster unsupported: If the cluster Prometheus should be configured to scrape metrics")
 	cmd.Flags().BoolVar(&opts.EnableCIDebugOutput, "enable-ci-debug-output", opts.EnableCIDebugOutput, "If extra CI debug output should be enabled")
+	cmd.Flags().StringVar(&opts.AWSCreds, "aws-creds", opts.AWSCreds, "Path to an AWS credentials file (AWS only)")
```

Comment:
If this is now required, maybe [the docs](https://hypershift-docs.netlify.app/quickstart/#create-a-cluster) should be updated to reflect the new requirement? 😁

Reply:
I see that it's not required here... when _is_ it required?

Reply:
Shouldn't this default to then value of `AWS_SHARED_CREDENTIALS_FILE`?

Reply:
@ironcladlou it is not required.  Updated the PR description better explaining this.

@alvaroaleman a good idea, but a little too magical I think.  The user might have an over privileged cred set in their environment and we don't want use that implicitly IMHO.

Reply:
>The user might have an over privileged cred set in their environment and we don't want use that implicitly IMHO.

Good point, I think this convinces me explicit is safer